src/containers/App/SearchPage.tsx

```diff
@@ -13,7 +13,7 @@ import { Route, Routes } from "react-router-dom";
 import { UseQueryResult } from "@tanstack/react-query";
 import { List } from "@ndla/icons/action";
 import { SearchMedia, SearchContent, Concept, SquareAudio } from "@ndla/icons/editor";
-import Footer from "./components/Footer";
+import Footer from "./components/FooterWrapper";
```

Comment:
Må vel endre denne også.
```suggestion
import FooterWrapper from "./components/FooterWrapper";
```

Reply:
Eller er kanskje ikkje så farlig

Reply:
Jeg kan gjøre det, men da må jeg egentlig bytte alle de andre stedene også